{"level":"info","message":"Connected to mongodb://localhost/vidly..."}
{"body":"{\n\t\"Horror\"\n}","expose":true,"level":"error","message":"Unexpected token } in JSON at position 12 Unexpected token } in JSON at position 12","stack":"SyntaxError: Unexpected token } in JSON at position 12\n    at JSON.parse (<anonymous>)\n    at parse (C:\\Users\\user\\vidly\\node_modules\\body-parser\\lib\\types\\json.js:89:19)\n    at C:\\Users\\user\\vidly\\node_modules\\body-parser\\lib\\read.js:121:18\n    at invokeCallback (C:\\Users\\user\\vidly\\node_modules\\raw-body\\index.js:224:16)\n    at done (C:\\Users\\user\\vidly\\node_modules\\raw-body\\index.js:213:7)\n    at IncomingMessage.onEnd (C:\\Users\\user\\vidly\\node_modules\\raw-body\\index.js:273:7)\n    at IncomingMessage.emit (node:events:402:35)\n    at endReadableNT (node:internal/streams/readable:1343:12)\n    at processTicksAndRejections (node:internal/process/task_queues:83:21)","status":400,"statusCode":400,"type":"entity.parse.failed"}
{"body":"{\n\t\"id\": 4\n\t\"name\": \"Horror\"\n}","expose":true,"level":"error","message":"Unexpected string in JSON at position 12 Unexpected string in JSON at position 12","stack":"SyntaxError: Unexpected string in JSON at position 12\n    at JSON.parse (<anonymous>)\n    at parse (C:\\Users\\user\\vidly\\node_modules\\body-parser\\lib\\types\\json.js:89:19)\n    at C:\\Users\\user\\vidly\\node_modules\\body-parser\\lib\\read.js:121:18\n    at invokeCallback (C:\\Users\\user\\vidly\\node_modules\\raw-body\\index.js:224:16)\n    at done (C:\\Users\\user\\vidly\\node_modules\\raw-body\\index.js:213:7)\n    at IncomingMessage.onEnd (C:\\Users\\user\\vidly\\node_modules\\raw-body\\index.js:273:7)\n    at IncomingMessage.emit (node:events:402:35)\n    at endReadableNT (node:internal/streams/readable:1343:12)\n    at processTicksAndRejections (node:internal/process/task_queues:83:21)","status":400,"statusCode":400,"type":"entity.parse.failed"}
{"level":"info","message":"Connected to mongodb://localhost/vidly..."}
{"body":"{\n\t\"_id\": \"123\"\n\t\"name\": \"Horror\"\n}","expose":true,"level":"error","message":"Unexpected string in JSON at position 17 Unexpected string in JSON at position 17","stack":"SyntaxError: Unexpected string in JSON at position 17\n    at JSON.parse (<anonymous>)\n    at parse (C:\\Users\\user\\vidly\\node_modules\\body-parser\\lib\\types\\json.js:89:19)\n    at C:\\Users\\user\\vidly\\node_modules\\body-parser\\lib\\read.js:121:18\n    at invokeCallback (C:\\Users\\user\\vidly\\node_modules\\raw-body\\index.js:224:16)\n    at done (C:\\Users\\user\\vidly\\node_modules\\raw-body\\index.js:213:7)\n    at IncomingMessage.onEnd (C:\\Users\\user\\vidly\\node_modules\\raw-body\\index.js:273:7)\n    at IncomingMessage.emit (node:events:402:35)\n    at endReadableNT (node:internal/streams/readable:1343:12)\n    at processTicksAndRejections (node:internal/process/task_queues:83:21)","status":400,"statusCode":400,"type":"entity.parse.failed"}
{"body":"{\n\t\"email\": \"notnor24@walla.com\"\n\t\"password\": \"12345\"\n}","expose":true,"level":"error","message":"Unexpected string in JSON at position 34 Unexpected string in JSON at position 34","stack":"SyntaxError: Unexpected string in JSON at position 34\n    at JSON.parse (<anonymous>)\n    at parse (C:\\Users\\user\\vidly\\node_modules\\body-parser\\lib\\types\\json.js:89:19)\n    at C:\\Users\\user\\vidly\\node_modules\\body-parser\\lib\\read.js:121:18\n    at invokeCallback (C:\\Users\\user\\vidly\\node_modules\\raw-body\\index.js:224:16)\n    at done (C:\\Users\\user\\vidly\\node_modules\\raw-body\\index.js:213:7)\n    at IncomingMessage.onEnd (C:\\Users\\user\\vidly\\node_modules\\raw-body\\index.js:273:7)\n    at IncomingMessage.emit (node:events:402:35)\n    at endReadableNT (node:internal/streams/readable:1343:12)\n    at processTicksAndRejections (node:internal/process/task_queues:83:21)","status":400,"statusCode":400,"type":"entity.parse.failed"}
